database/dbHelpers.js

```diff
@@ -1,4 +1,6 @@
 
+const db = require('../database/index');
```

Comment:
This is irrelevant to this particular commit, but I'm wondering how you're able to get info from all three tables without an explicit JOIN statement